src/core/stdc/stdlib.d

```diff
@@ -153,6 +153,13 @@ else
     void    srand(uint seed);
 }
 
+version (CRuntime_Glibc)
+{
+    version = AlignedAllocSupported;
+}
+// Add other C runtimes as necessary...
+else {}
+
```

Comment:
On second look, I think it's kind of awkward to have this version block sitting there middle. It should either be in the beginning of the file (e.g. after the `Darwin` version block) or right before the only use of `AlignedAllocSupported`.